### PR TITLE
Replaced Cookie Storage with Local Storage (localforage)

### PR DESCRIPTION
**Pull request by clloyd**
This replaces the cookie store with localforage (https://github.com/mozilla/localForage) based storage, will store the last 100 enabled urls (can be increased/decreased)

### CODE REVIEW

File: public/config.js [hunk 1]
```diff
@@ -18,6 +43,7 @@ System.config({
     "composer-components": "github:guardian/tools-components@1.0.8",
     "core-js": "npm:core-js@1.1.1",
     "css": "github:systemjs/plugin-css@0.1.13",
+    "localforage": "npm:localforage@1.2.10",
     "github:angular/bower-angular-route@1.4.3": {
```
Comment (jamespamplin) [inline public/config.js]: Noticed that angular is being included here. Are we using Angular in viewer? Can't seem to find it.

File: public/config.js [hunk 2]
```diff
@@ -11,6 +11,31 @@ System.config({
     "github:*": "jspm_packages/github/*",
     "npm:*": "jspm_packages/npm/*"
   },
+  bundles: {
```
Comment (jamespamplin) [inline public/config.js]: Should the bundle config be checked in?

**Comment by jamespamplin** [comment]
:+1: few comments, but looks good! Especially like use of localforage.

File: public/config.js (hunk 1 above)
Comment (clloyd) [inline public/config.js]: Just an outdated jspm map, from a test I did (wish it removed these!)

**Pull request merged**